src/Chirp.Web/CheepService.cs

```diff
@@ -21,12 +21,10 @@ public class CheepService : ICheepService
     private readonly IAuthorRepository _Author;
     private readonly ICheepRepository _Cheep;
```

Comment:
Naming could be more descriptive, as "Cheep" and "Author" in this case references repositories, whereas the same names references  actual cheeps and authors elsewhere.

 Recommended naming is "_AuthorRepo" and "_CheepRepo"